packages/loader/container-loader/src/test/connectionStateHandler.spec.ts

```diff
@@ -1139,6 +1143,48 @@ describe("ConnectionStateHandler Tests", () => {
 		},
 	);
 
+	it("test 'read' reconnect & races ", async () => {
+		connectionStateHandler = createHandler(
+			false, // connectedRaisedWhenCaughtUp,
```

Comment:
Does the test require this?  Better to use the default values if possible

Reply:
It does - I pass readClientsWaitForJoinSignal = true, while default uses false for both arguments

Reply:
Looks like they default to true to me.  In `createConnectionStateHandler`, for each it is looking for a "Disable" flag, and if not set will yield true.